tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest keeping the encoding parameters on separate lines, to improve readability, similar to earlier code blocks.</p><p> </p><p><code># Define a line plot by reducing intra-annual observations to median.</code></p><p><code>line = alt.Chart(climate_df).mark_line().encode(</code></p><p><code>    x='Year:O',</code></p><p><code>    y=alt.Y('median(Precip-mm):Q', title='Precipitation (mm/month)'),</code></p><p><code>    color='Model')</code></p><p> </p><p><code># Define an interquartile range plot to describe the intra-annual distribution.</code></p><p><code>band = alt.Chart(climate_df).mark_errorband(extent='iqr').encode(</code></p><p><code>    x='Year:O',</code></p><p><code>    y=alt.Y('Precip-mm:Q', title='Precipitation (mm/month)'),</code></p><p><code>    color='Model')</code></p><p> </p><p><code># Combine the band and line plots; display it.</code></p><p><code>(band + line).properties(width=600, height=300)</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='134'/>

Reply:
<p>Done.</p>